### DISCUSSION TITLE
Popover Forces Scrolling to Bottom of Page

### DISCUSSION THREAD

**Discussion by tgaeta** [tailwindlabs/headlessui#2298] [Help]
### Problem
Hey all, I've checked the solved/open issues for scrolling issues pertaining to using a Popover. None of those issues seemed to reveal a solution on how to mitigate the scrolling when a popover is opened. I have tried several different approaches from event listeners, stop propagation, useEffect to check if the popover is open, and then stop scrolling, etc. I've attached a video to demonstrate what is occurring.

I even tried upgrading tailwind to see if it would fix the issue but no luck. Can anyone think of a possible solution? Should I be using a different HeadlessUI component for this scenario?

#### Current Versions
```json
"dependencies": {
  "tailwindcss": "3.0.24",
  "@headlessui/react": "1.7.2",
  "next": "12.1.4",
}
  ```
  
### Demo

https://user-images.githubusercontent.com/9040988/220426035-b8e98207-c724-4021-bceb-4bd315903fe5.mov

### Sample Code

```tsx
return (
  <Popover className={cx('lg:hidden', popoverClassName)} data-testid={dataTestId} id="popover">
    {({ open }) => (
      <>
        {!open && <Popover.Button>{button}</Popover.Button>}
        <AnimatePresence initial={false}>
          {open && (
            <>
              <Popover.Overlay
                static
                key="overlay"
                className="bg-secondary-900/30 fixed inset-0 backdrop-blur-[2px]"
                {...overlayMotion}
              />
              <Popover.Panel static key="panel" className={customPanelClassName} {...panelMotion}>
                {children}
              </Popover.Panel>
            </>
          )}
        </AnimatePresence>
      </>
    )}
  </Popover>
);
```

*Update
When I test this in iOS Simulator, the page doesn't scroll (as expected). This appears to occur in the browser when the viewport is at mobile size. Tested in Safari, Brave, Chrome, and Firefox.

**Comment by chrisjansky** [comment]
Hi, I have a related issue: https://github.com/tailwindlabs/headlessui/discussions/2388

Did you manage to find a fix? 🙏
